project1/styles.css

```
@@ -0,0 +1,49 @@
+body {
+    background-color: rgba(236, 236, 236, 0.226)
```

In addition to `background-color`, I recommend always adding a font and text color to the `body` element. I'd use whatever represents _most_ of the text and then alter that with your later CSS, e.g. :
```suggestion
    background-color: rgba(236, 236, 236, 0.226);
    color: rgba(83, 83, 83, 0.801);
    font-family: 'Lato', sans-serif;
```